cms/envs/aws.py

```diff
@@ -357,3 +357,20 @@
 
 PROCTORING_BACKEND_PROVIDER = AUTH_TOKENS.get("PROCTORING_BACKEND_PROVIDER", PROCTORING_BACKEND_PROVIDER)
 PROCTORING_SETTINGS = ENV_TOKENS.get("PROCTORING_SETTINGS", PROCTORING_SETTINGS)
+
+##### Third-party auth options ################################################
+if FEATURES.get('ENABLE_THIRD_PARTY_AUTH'):
+    AUTHENTICATION_BACKENDS = (
+        ENV_TOKENS.get('THIRD_PARTY_AUTH_BACKENDS', [
```

Comment:
@maxrothman what do you think of having this setting synced between LMS/CMS via the configuration repo? Would it be possible to have a common setting which is written to both the cms and lms env files, and do you think it owuld make sense in this case? Pinging you since it would require coordination with devops.